node-launchpad/.config/config.json5

```diff
@@ -1,48 +1,73 @@
 {
   "keybindings": {
-    "Home": {
-      "<Ctrl-s>": {"HomeActions":"StartNodes"},
-      "<Ctrl-S>": {"HomeActions":"StartNodes"},
-      "<Ctrl-Shift-s>": {"HomeActions":"StartNodes"},
-      "<Ctrl-x>": {"HomeActions":"StopNodes"}, 
-      "<Ctrl-X>": {"HomeActions":"StopNodes"},
-      "<Ctrl-Shift-x>": {"HomeActions":"StopNodes"},
-
-      "up" : {"HomeActions":"PreviousTableItem"},
-      "down": {"HomeActions":"NextTableItem"},
-
-      "tab": {"TabActions":"NextTab"},
-      "<Shift-tab>": {"TabActions":"PreviousTab"},
-
-      "<Ctrl-b>": {"HomeActions":"TriggerBetaProgramme"},
-      "<Ctrl-B>": {"HomeActions":"TriggerBetaProgramme"},
-      "<Ctrl-Shift-b>": {"HomeActions":"TriggerBetaProgramme"},
-      "<Ctrl-g>": {"HomeActions":"TriggerManageNodes"},
-      "<Ctrl-G>": {"HomeActions":"TriggerManageNodes"},
-      "<Ctrl-Shift-g>": {"HomeActions":"TriggerManageNodes"},
-      "<h>": {"HomeActions":"TriggerHelp"},
-      "<H>": {"HomeActions":"TriggerHelp"},
-      "<Shift-h>": {"HomeActions":"TriggerHelp"},
-      "<Ctrl-r>": {"HomeActions":"TriggerResetNodesPopUp"},
-      "<Ctrl-R>": {"HomeActions":"TriggerResetNodesPopUp"},
-      "<Ctrl-Shift-r>": {"HomeActions":"TriggerResetNodesPopUp"},
+    "Status" : {
+      "<s>": {"MenuActions":"StatusTab"},
+      "<S>": {"MenuActions":"StatusTab"},
+      "<o>": {"MenuActions":"OptionsTab"},
+      "<O>": {"MenuActions":"OptionsTab"},
+      "<h>": {"MenuActions":"HelpTab"},
+      "<H>": {"MenuActions":"HelpTab"},
```

Comment:
Can we not directly call `Action::SwitchScene` here? The `MenuActions` just trigger swtich scene everywhere, so why don't we do it directly here?

Reply:
Similary for `Options` and `Help` screens